src/App/DataSource/Twitter/Twitter.php

```diff
@@ -71,9 +71,10 @@ public function getOptions()
     {
         return [
             'intro_step1' => [
-                'label' => 'Step 1: Create a new Twitter application',
+                'label' => 'Step 1: Create a new Twitter application.<br /> "Twitter applications may take a few hours or a few days to be approved by Twitter. <br />Please be aware of this if you need this data quickly.',
```

Comment:
@Jenniline Thank you for making this change! Could you just remove the " before Twitter? Thank you 🎉

Reply:
Alright. I am on it